### PR TITLE
Made the revisions we spoke of earlier this week.

### PR DESCRIPTION
**Pull request by BrianVales**
I made the revisions we spoke of. Formatted FTL's, JS, and SCSS files. Wrote some JS logic to fix the banner when the menu is out.

You will need to add the images to each web content portion after bringing in the theme due to the bug we are experiencing. Also, there is a lar file in the resources importer to import into the blogs.

### CODE REVIEW

File: modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="about-wrapper">
+	<div class="about-flex">
+		<div class="about-us">
+
+			<h2>${AboutUsTitle.getData()}</h2>
+
+			<p>${AboutUsDescription1.getData()}
```
[the remark this answers is not in the document]
Comment (BrianVales) [reply modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl]: If I am understanding correctly you are speaking about the camelCasing?
```for example
			<p>${AboutUsDescription1.getData()}
```

```should be
			<p>${about-us-description-1.getData()}
```

please advise if this is correct.

File: modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+					</div>
+
+					<div class="footer-social">
+
+						<h3 class="h3">Stay Connected</h3>
+
+						<ul class="carousel-ul" id="footer-count">
+
+							<#if show_facebook>
+								<li class="carousel-social" title="Facebook"><a class="fa-facebook-f fab" href="${facebook_url}"></a></li>
```
[the remark this answers is not in the document]
Comment (BrianVales) [reply modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl]:
```suggestion
								<li class="carousel-social" title="Facebook"><a class="fa-facebook-f fab" href="${facebook_url}"></a></li>
```

I'm not sure what I am hardcoding in this instance. Can you please clarify so I can fix it?

File: modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonmak08) [reply modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl]: No, we were not talking about camelCasing of the variable.

File: modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonmak08) [reply modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl]: The word `Facebook` in the title tag is hardcoded no?  It is not referencing a language key.

File: modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl (hunk 1 above)
[the remark this answers is not in the document]
Comment (BrianVales) [reply modules/apps/frontend-theme/real-business-theme/src/WEB-INF/src/resources-importer/journal/templates/about-descriptor/about-descriptor.ftl]: the p tags. I have taken care of them.

File: modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl (hunk 2 above)
[the remark this answers is not in the document]
Comment (BrianVales) [reply modules/apps/frontend-theme/real-business-theme/src/templates/portal_normal.ftl]: Ok. Thank you for the clarification.

**Pull request closed without merging**